settings_test.py

```diff
@@ -6,6 +6,7 @@
 INSTALLED_APPS += (
     'olympia.translations.tests.testapp',
     'olympia.core.tests.db_tests_testapp',
+    'olympia.core.tests.m2m_testapp',
```

Comment:
Couldn't put the testapp under `amo` as it caused weird initialisation errors during the test setup.